genai/client_test.go

```diff
@@ -122,7 +122,7 @@ func TestLive(t *testing.T) {
 
 		checkMatch(t,
 			send("Name puppy breeds.", false),
-			"Beagle", "Poodle")
```

Comment:
Nit: you'll probably want both and a few others to reduce the chance of flakiness

Also phrasing the question as "Name the 5 most popular puppy breeds" reduces entropy